Header header
float32 kpPosXY
float32 kpPosZ
float32 KiPosZ
float32 kpVelXY
float32 kpVelZ
float32 kpBank
float32 kpYaw
float32 kpPQR_x
float32 kpPQR_y
float32 kpPQR_z
float32 Traj_x
float32 Traj_y
float32 Traj_z
float32 Traj_vx
float32 Traj_vy
float32 Traj_vz
float32 Traj_yaw